net/net_demo_installer

```diff
@@ -145,8 +145,12 @@ function sudoExec() {
     sudo -E $@
 }
 
-contiv_network_version="v0.1-05-19-2016.08-34-56.UTC"
+: ${contiv_network_version:=v0.1-05-19-2016.08-34-56.UTC}
+echo "Using version: $contiv_network_version"
+
 docker_version="1.11.1"
+
+#docker_version="1.11.1"
```

Comment:
why comment out 1.11.1 version? do you want to change something else to default?